distro/adaptation/centos

```diff
@@ -12,9 +12,10 @@ cgroupfs-mount:
 clang-6.0: clang.x86_64
 cmake: cmake cmake3
 cron: cronie
-dbench=4.0-2: dbench
+dbench=4.0-2:
```

Comment:
let's keep dbench mapping for now, it probably uses lkp-tests/programs/dbench

Reply:
ok!